BookReader/BookReader.css

```diff
@@ -510,6 +510,7 @@ body.BRfullscreenActive .mm-menu {
 }
 .BRpageview {
   position: relative;
+  margin: auto;
```

Comment:
`BookReader.css` is a generated file, so this change should also be made in the corresponding `.scss` file.

Reply:
thanks!